drivers/gke/gke_driver.go

```diff
@@ -288,14 +378,27 @@ func (d *Driver) GetDriverUpdateOptions(ctx context.Context) (*types.DriverFlags
 
 // SetDriverOptions implements driver interface
 func getStateFromOpts(driverOptions *types.DriverOptions) (state, error) {
+	var cbs []*raw.CidrBlock
```

Comment:
This variable is not necessary as the only occurrence is for cidrblocks initialization.